skycultures/almagest/constellation_names.eng.fab

```diff
@@ -1,27 +1,27 @@
 UMi	"Μικρή ʾˊΑρκτος (Mikre Arktos)"	_("Small Bear")
 UMa	"Μεγαλή ʾˊΑρκτος (Megale Arktos)"	_("Great Bear")
-Dra	"Δράκοντος (Drakontos)"		_("Dragon")
+Dra	"Δράκων (Drakun)"		_("Dragon")
 Cep	"Κηφέως (Kepheus)"	_("King Kepheus")
```

Comment:
Is it not "-eus" in Greek?

Reply:
δράκος = Eye 
δράκων = a dragon (possible form according to Middle Liddell: δρακών, δράκων, sg,masc, nom)